cirq/ops/__init__.py

```diff
@@ -58,6 +58,9 @@
     CZPowGate,
     H,
     HPowGate,
+    Rx,
+    Ry,
+    Rz,
     rx,
```

Comment:
I think we can deprecate these in favor for the R* constructors.

Reply:
Filed an issue to track this: #3929